samples/customEP2/custom_ep2.cc

```diff
@@ -0,0 +1,150 @@
+#include <memory>
+#include <cmath>
+#include <iostream>
+#include "custom_ep2.h"
+#include "core/session/onnxruntime_lite_custom_op.h"
+#include "core/session/onnxruntime_cxx_api.h"
+#include "core/framework/ortdevice.h"
+#include "core/framework/ortmemoryinfo.h"
+
+namespace onnxruntime {
+
+/////////////////////////////////////// POC Kernels ////////////////////////////////////////////
+onnxruntime::Status Identity(onnxruntime::interface::TensorView<float>& input,
+                             onnxruntime::interface::Tensor<float>& output) {
+  const auto& shape = input.Shape();
+  const float* input_data = input.Data();
+  float* output_data = output.Allocate(shape);
+  size_t number_of_elements = input.NumberOfElements();
+  for (size_t i = 0; i < number_of_elements; ++i) {
+      output_data[i] = input_data[i];
+  }
+  return onnxruntime::Status::OK();
+}
+struct Celu {
+  Celu(const interface::IKernelInfo&) {}
+  onnxruntime::Status Compute(onnxruntime::interface::TensorView<float>& input,
+                              onnxruntime::interface::Tensor<float>& output) {
```

Comment:
Should we use the interface (interface::ITensor) or the implementation (interface::Tensor) as the parameter?